lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -168,6 +168,33 @@ L<Mojolicious::Command::get>.
 
   $ ./myapp.pl get -v -M POST -c 'test' /echo
 
+=head2 JSON Services
+
+JSON is a very common interchange format for web services. Mojolicious loves
+JSON and makes it very easy to both build and use JSON services. In this example
+you can fetch module data from L<MetaCPAN|http://metacpan.org>.
```

Comment:
Maybe it would be better to write 'In this example we fetch module data from ...'?